GNUmakefile

```diff
@@ -130,3 +130,8 @@ link-git-hooks: ## Install git hooks
 	@echo "==> Installing all git hooks..."
 	find .git/hooks -type l -exec rm {} \;
 	find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;
+
+.PHONY: update-atlas-sdk
+update-atlas-sdk: ## Update the atlas-sdk dependency
+	go install github.com/icholy/gomajor@v0.9.5
+	gomajor get go.mongodb.org/atlas-sdk/v20230201001@latest
```

Comment:
question: would the version `v20230201001` have to be modified manually or will it always fetch the latest major version?

Reply:
My understanding is that it should change for every new release version.
If I am correct, could we update the command to use an env variable for instance `ATLAS_SDK_VERSION=v20230201001` and update  CONTRIBUTING.md to include an explanation that this var must contain the version of the SDK that we want to add to TF

Reply:
> would the version v20230201001 have to be modified manually 

No. 

>  or will it always fetch the latest major version?

yes

TL;DR:

Someone back in 2009 make call that:

`get go.mongodb.org/atlas-sdk` always refers to 0.x.x and 1.x.x versions.

anything above needs:

get go.mongodb.org/atlas-sdk/someversion` - version here is irrelevant but need to exist.